tests/cacl/test_cacl_application.py

```diff
@@ -835,6 +835,98 @@ def generate_scale_rules(duthost, ip_type):
     duthost.command("iptables -I INPUT 3 -p tcp -m tcp --dport 22 -j ACCEPT")
 
 
+def verify_cacl_show_acl_rule(duthost):
```

Comment:
@matthew-soulsby  could you pls check if this function can check DROP rules? please see the issue here:
https://github.com/sonic-net/sonic-mgmt/issues/10426

Reply:
@ZhaohuiS Had a look at the traceback for my last test run, and it appears that DROP rules are in the expected rules, checked against the actual rules, and they pass the configuration check:

```
actual_rules = {'RULE_1': {'SRC_IP': '20.0.0.2/32', 'action': 'DROP', 'name': 'RULE_1 ', 'priority': '9999'}, 'RULE_10': {'SRC_IP': '...riority': '9989'}, 'RULE_12': {'SRC_IP': '20.0.0.13/32', 'action': 'DROP', 'name': 'RULE_12', 'priority': '9988'}, ...}
expected_rules = {'RULE_1': {'actions': {'config': {'forwarding-action': 'DROP'}}, 'config': {'sequence-id': 1}, 'ip': {'config': {'sou...arding-action': 'DROP'}}, 'config': {'sequence-id': 12}, 'ip': {'config': {'source-ip-address': '20.0.0.13/32'}}}, ...}
```

How does that look to you?